components/PageControl/src/MDCPageControl.m

```diff
@@ -56,6 +58,16 @@ static inline CGFloat normalizeValue(CGFloat value, CGFloat minRange, CGFloat ma
   return (diff > 0) ? ((value - minRange) / diff) : 0;
 }
 
+/**
+ Snap @c value to the range <code>[min, max]</code>.
+ */
+static inline NSInteger snapToRange(NSInteger min, NSInteger max, NSInteger value) {
```

Comment:
Functions are typically capitalized.
For minimal surprise, consider naming Clamp(value, min, max) to match other implementations.